source/cluster-config-file.txt

```diff
@@ -0,0 +1,161 @@
+.. _atlas-cli-cluster-config-file:
+
+==============================
+{+Cluster+} Configuration File
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use a {+cluster+} configuration file to specify the required settings when you create or update a {+cluster+} using the 
+{+atlas-cli+}. The {+atlas-cli+} accepts ``.json``
+{+cluster+} configuration files.
+
+Use the resources below to:
+
+- Learn the :ref:`required settings <required-cluster-settings>` you
+  can specify in the {+cluster+} configuration file.
+- View and copy :ref:`sample configuration files
+  <example-cluster-config-file>`.
+
+.. _required-cluster-settings:
+
+Required {+Cluster+} Settings
+-----------------------------
+
+|service| requires the following settings to create a {+cluster+} with
+the {+atlas-cli+}. You must specify these {+cluster+} settings either
+in the configuration file or as flags in the command:
+
+.. list-table:: 
+   :header-rows: 1 
+   :widths: 20 10 70 
+
+   * - Field 
+     - Type 
+     - Description 
+
+   * - ``clusterType``
+     - string
+     - Type of {+cluster+} that you want to create. Valid values are
+       ``REPLICASET``, ``SHARDED``, or ``GEOSHARDED``.
+
+   * - ``name``
+     - string
+     - Human-readable label that identifies the {+cluster+}.
+
+   * - ``replicationSpecs``
+     - array
+     - List containing the configuration settings for {+cluster+}
+       regions and the hardware provisioned in them.
```

Comment:
**Style:** See [Clarify Gerunds and Participles](https://www.mongodb.com/docs/meta/style-guide/writing/clarify-gerunds-participles/)
```suggestion
     - List that contains the configuration settings for your {+cluster+}
       regions and the hardware provisioned in them.
```